### DISCUSSION TITLE
Student is not getting added

### DISCUSSION THREAD

**Discussion by rahulpro2001** [TEAMMATES/teammates#12680] [Help requests]
hii after sending invite through Instructor page how can I join the student because there is no joining link for the Student and also not getting mail on the student Email id.

**Comment by weiquu** [comment]
Hi @rahulpro2001, here are the steps:
1. Log in as the instructor of the course you want to add the student to
2. Add the student to the course via the enrol page
3. Log out from the instructor account and [log in as admin](https://teammates.github.io/teammates/development.html#logging-in-to-a-teammates-instance)
4. Click on search and search for that student
5. Expand the tab below (by clicking on the arrow down button) and copy the course join link
6. Log out of the admin account
7. Go to the copied link
8. Log in using the student's email address
9. Press register for course

Let me know if this works, or if you need guiding screenshots
